Codegen/test/resources/output/TypeCobol/ProcedureCall-Public.rdz.tcbl

```diff
@@ -116,11 +118,16 @@
       * __________________________________________________
       * OK : proper parameter list (TCRFUN_CALL_PRIVATE_ONLY_SAME_PGM)
       *    CALL Pgm1PrivateValidateDateFormat
-      *             INPUT      somedate someformat
+      *             INPUT      somedate
+      *                        someformat
+      *                        length of somedate
+      *                        length of somedate::YYYY
```

Comment:
The code generated here is wrong, it should be:
length of YYYY of somedate (or YYYY IN somedate)